skada/tests/test_pipeline.py

```diff
@@ -135,13 +135,17 @@ def test_named_estimator():
         ('adapter', SubspaceAlignmentAdapter(n_components=2)),
         PCA(n_components=4),
         PCA(n_components=2),
+        CORAL(),
```

Comment:
```suggestion
        CORALAdapter(),
```

Reply:
This is just to test names. I use `CORAL` here twice to make sure we properly rename the nested step when the name for the pipeline is provided. In short, I need this to a be a DA pipeline not an adapter.

Reply:
OK good point